packages/components/components/elvis-popover/src/react/useFocusTrap.ts

```diff
@@ -0,0 +1,16 @@
+import { useFocusTrap as useFocusTrapToolbox } from '@elvia/elvis-toolbox';
+import { RefObject, useEffect } from 'react';
+
+export const useFocusTrap = (trapRef: RefObject<HTMLElement>, isOpen: boolean) => {
+  const { trapFocus, releaseFocusTrap } = useFocusTrapToolbox();
+
+  useEffect(() => {
+    if (!isOpen) {
+      return;
+    }
+    trapFocus(trapRef);
+    return () => {
+      releaseFocusTrap();
+    };
+  }, [isOpen]);
```

Comment:
I like these small, focused hooks! This feels like something that the `useFocusTrap` from the toolbox could deal with. Would it make sense to provide a `trapRef` and `isOpen` to the `useFocusTrap` from the toolbox, and then simply call it as a method in the components that use the focus trap?

Reply:
Yeah, it definitely would! However that would be a breaking change in the toolbox, so it would have to be a new name to avoid that 😅

Reply:
Mm lets skip it for now